tests/test-files/transform/Elm-0.18/Examples.formatted.elm

```diff
@@ -2,31 +2,15 @@ module Main exposing (bar, multilineList, ratio)
 
 
 ratio =
-    graphHeight
-        / (if range == 0 then
-            0.1
-
-           else
-            toFloat range
-          )
+    graphHeight / (if range == 0 then 0.1 else toFloat range)
 
 
 
 -- foo=(case x of {True->1;False->3})
 
 
 bar =
-    if
-        if a then
-            True
-
-        else
-            False
-    then
-        "a"
-
-    else
-        "b"
+    if if a then True else False then "a" else "b"
```

Comment:
Do we want this?

Reply:
I've never written code like that and have never seen any write code like that in reviews. Maybe because it's not very idiomatic functional code. I'm not against it, although I'd say it's a little less readable compared to the multiline version. 

If we want to discourage code like this by having it format in a certain way, I'm not sure if one or the other does a better job at that.